retromock/build.gradle

```diff
@@ -4,7 +4,8 @@ group = 'co.infinum'
 version = "$versions.retromock"
 
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    apply from: 'versions.gradle'
```

Comment:
We already apply this in the first line. Is it really needed here as well?

Reply:
Unfortunately, it only works if I apply it on both places.

Reply:
That was incorrect, build passes also if apply is only in the buildscript block.